src/couch/src/couch_changes.erl

```diff
@@ -368,6 +395,17 @@ open_revs(Db, DocInfo, Style) ->
     OpenResults = [couch_db:open_doc(Db, DI, OpenOpts) || DI <- DocInfos],
     [Doc || {ok, Doc} <- OpenResults].
 
+open_revs2(Db, DocInfo) ->
```

Comment:
Wonder if we should rename this to be a bit more descriptive: `open_all_revs_include_doc`?